### ISSUE TITLE
Oxygen Blocks for Gutenberg - Edit Div Background Image

### ISSUE THREAD

**Issue by yepyeh** [soflyy/oxygen-bugs-and-features#1922]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*
I would like to create a Block with a div with a background image. For example a 2 column block, text on the left and div with background image on the right. I would like the user to be able to edit the image in the Gutenberg editor.

**What are the use cases for this feature?**
*e.g. Accessibility, e-commerce store that sells bicycles, etc.*
Gutenberg blocks. Text on Left, Image on Right and the other way around.
![Screenshot 2021-02-19 at 15 50 53](https://user-images.githubusercontent.com/2827763/108481086-4e723580-72ca-11eb-8eee-871a5c89f245.png)

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*

**Comment by kunalllll** [comment]
Ya That will be grate to present to Clint

**Comment by ChrisGV04** [comment]
Yeah. I haven't seen any page builder do that (to my knowledge) so it would be amazing to have it in Oxygen!

**Comment by jasonwilliamsau** [comment]
This would help address the issues I'm having in this bug too! https://github.com/soflyy/oxygen-bugs-and-features/issues/1920
